spinta/backends/__init__.py

```diff
@@ -192,7 +192,8 @@ def simple_data_check(
     backend: Backend,
     value: object,
 ) -> None:
-    check_type_value(dtype, value)
+    if data.action in (Action.UPDATE, Action.INSERT):
+        check_type_value(dtype, value)
```

Comment:
Could you please change `raise Exception` in `check_type_value` into a new exception from `spinta/exceptions.py`, extended from `UserError`? These exceptions provides a lot more information to the client.